### PR TITLE
Suggested namechanges

### PR DESCRIPTION
**Pull request by matozoid**
I've kind of forgotten what it was I was still going to do, but I bet it all comes back if I just start with a simple thing like this suggested name change.

I'm kind of worried that no unit test was touched for this.

### CODE REVIEW

**Comment by matozoid** [comment]
image

File: src/main/kotlin/com/strumenta/kolasu/model/Processing.kt [hunk 1]
```diff
@@ -162,19 +162,19 @@ val Node.children: List<Node>
     }
 
 // TODO reimplement using transformChildren
```
Comment (ftomassetti) [inline src/main/kotlin/com/strumenta/kolasu/model/Processing.kt:164]: We should also update this comment now, I think

**Comment by ftomassetti** [comment]
Thank you!

File: src/main/kotlin/com/strumenta/kolasu/model/Processing.kt (hunk 1 above)
Comment (matozoid) [reply src/main/kotlin/com/strumenta/kolasu/model/Processing.kt:164]: I'll do that in the next PR.

**Pull request merged**